### PR TITLE
Marketplace NC-672 Feature Overhaul

### PR DESCRIPTION
**Pull request by Joeper214**

### CODE REVIEW

File: app/controllers/app_settings_controller.rb [hunk 1]
```diff
@@ -122,11 +142,14 @@ def chef_create
       CfTemplate
       # rubocop:enable Lint/Void
 
+      cf_params = Rails.cache.fetch(CF_PARAMS_KEY) || {}
+      Rails.cache.delete(CF_PARAMS_KEY)
+
```
Comment (skyarch-networks) [inline app/controllers/app_settings_controller.rb]: __[rubocop]__
* Metrics/BlockLength: Block has too many lines. [43/25]

**Comment by skyarch-networks** [comment]
Click here if you want to create rubocop's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/31754199/issues/pull_request/399/diff/eb0f5801065656da4d87486d1bf7e6dc39dcc301?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by Joeper214** [comment]
Please Review and Merge PR @root-beer  🙇

**Comment by root-beer** [comment]
LGTM! Thanks. :bow:

**Comment by Joeper214** [comment]
ありがとうございます。

**Pull request merged**